src/libs/kata-types/src/capabilities.rs

```diff
@@ -17,6 +17,8 @@ pub enum CapabilityBits {
     MultiQueueSupport,
     /// hypervisor supports filesystem share
     FsSharingSupport,
+    /// hypervisor supports memory hotplug probe interface
+    GuestMemoryHotplugProbe,
```

Comment:
rename to `GuestMemoryHotplugSupport`, to keep the format consistent?

Reply:
That's different, GuestMemoryHotplugProbe the capability is used to notify the guest kernel about memory hotplug event

Reply:
I think this is enough for `qemu` and `dragonball`. Some hypervisor would rely on this capability, but some are not.